Formula/flint.rb

```diff
@@ -0,0 +1,56 @@
+class Flint < Formula
+  desc "C library for number theory"
+  homepage "http://flintlib.org"
+  url "http://flintlib.org/flint-2.5.2.tar.gz"
+  sha256 "cbf1fe0034533c53c5c41761017065f85207a1b770483e98b2392315f6575e87"
+  head "https://github.com/wbhart/flint2.git", :branch => "trunk"
+
+  depends_on "gmp"
```

Comment:
Are these runtime dependencies?

Reply:
Yeah, see test case where `libgmp` has to be linked.